specification/maps/resource-manager/Microsoft.Maps/preview/2023-12-01-preview/maps-management.json

```diff
@@ -929,9 +923,19 @@
               "Azure-AsyncOperation": {
                 "description": "Operation Status Location URI",
                 "type": "string"
+              },
+              "Location": {
+                "type": "string",
+                "description": "Operation Results Location URI"
               }
             }
           },
+          "200": {
+            "description": "Created the private endpoint connection properties successfully.",
```

Comment:
>Created [](http://example.com/codeflow?start=28&length=7)

Updated. This description was already correct. 200 is the PUT response when the resource already exists.